### ISSUE TITLE
Difference between session and academic term

### ISSUE THREAD

**Issue by jzthompson** [maltyeva/mindimp-pg#13]
Hi Maria,

I was wondering what the difference was between a "Session" and an "Academic Term" in the organization of courses on the website. I had been thinking of the course pilot as an academic term (a very short one), but it is listed as a session in the sample course in the "Sessions" section. I'm probably missing something, but to me the two terms seem to mean about the same thing.

**Comment by maltyeva** [comment]
Hi Jeff, 

Thanks for the question! So, in terms of what the language is on the website, I am completely open to changing it. I have a system on the backend that doesn't totally correspond to what the front-end looks like so I am open to any suggestions to improve readability. 

So in terms of academic sessions, a session is an offering of a term, for example the Spring 2015 Global Perspectives class, and you can have sessions like "Spring 2015 Global Perspectives A" or "Spring 2015 Global Perspectives B" for different levels as well. Right now I have the name pilot for the first session, but that can be changed. 

An academic term is a collection of courses that are offered that can have sessions of courses like global perspectives, english literature, poetry, etc grouped together. I added some test data to the website now to make it a little more clear, but let me know if you'd like to make any changes!
https://dry-citadel-6111.herokuapp.com/course_periods

Best, 
Maria

**Comment by jzthompson** [comment]
Ah, I see, so there could be different sessions of the same course within a single academic term. That makes sense, and I agree that "session" could be a useful distinction.

**Comment by maltyeva** [comment]
Yeah, and a term encompasses many different courses and subjects, sort of like having 3 sections of English 120 in the Fall 2015 semester.
